project-base/tests/FrontendApiBundle/Test/GraphQlTestCase.php

```diff
@@ -205,4 +235,124 @@ protected function assertResponseContainsArrayOfExtensionValidationErrors(array
         $this->assertArrayHasKey('validation', $errors[0]['extensions']);
         $this->assertIsArray($errors[0]['extensions']['validation']);
     }
+
+    /**
+     * @param array $expectedOrderItems
+     * @return \Shopsys\FrameworkBundle\Model\Pricing\Price
+     */
+    protected function getOrderTotalPriceByExpectedOrderItems(array $expectedOrderItems): Price
+    {
+        $totalPriceWithVat = Money::zero();
+        $totalPriceWithoutVat = Money::zero();
+
+        foreach ($expectedOrderItems as $expectedOrderItem) {
+            $expectedOrderItemTotalPrice = $expectedOrderItem['totalPrice'];
+            $totalPriceWithVat = $totalPriceWithVat->add(Money::create($expectedOrderItemTotalPrice['priceWithVat']));
+            $totalPriceWithoutVat = $totalPriceWithoutVat->add(
+                Money::create($expectedOrderItemTotalPrice['priceWithoutVat'])
+            );
+        }
+        return new Price($totalPriceWithoutVat, $totalPriceWithVat);
+    }
+
+    /**
+     * @param string $vatRate
+     * @return \Shopsys\FrameworkBundle\Model\Pricing\Vat\Vat
+     */
+    protected function getVatByVatRate(string $vatRate): Vat
+    {
+        $vats = $this->vatFacade->getAllForDomain($this->domain->getId());
+        foreach ($vats as $vat) {
+            if ((float)$vat->getPercent() === (float)$vatRate) {
+                return $vat;
+            }
+        }
+        throw new VatNotFoundException();
+    }
+
+    /**
+     * @param string $priceWithoutVat
+     * @param string $vatRate
+     * @param int $quantity
+     * @return \Shopsys\FrameworkBundle\Model\Pricing\Price
+     */
+    private function getConvertedPriceToDomainDefaultCurrency(
+        string $priceWithoutVat,
+        string $vatRate,
+        int $quantity = 1
+    ): Price {
+        $domainId = $this->domain->getId();
+        $vat = $this->getVatByVatRate($vatRate);
+        $currency = $this->currencyFacade->getDomainDefaultCurrencyByDomainId($domainId);
+
+        $basePrice = $this->basePriceCalculation->calculateBasePriceRoundedByCurrency(
+            $this->priceConverter->convertPriceWithoutVatToPriceInDomainDefaultCurrency(
+                Money::create($priceWithoutVat),
+                $domainId
+            ),
+            PricingSetting::INPUT_PRICE_TYPE_WITHOUT_VAT,
+            $vat,
+            $currency
+        );
+
+        return $this->basePriceCalculation->calculateBasePriceRoundedByCurrency(
+            $basePrice->getPriceWithVat()->multiply($quantity),
+            PricingSetting::INPUT_PRICE_TYPE_WITH_VAT,
+            $vat,
+            $currency
+        );
+    }
+
+    /**
+     * @param string $priceWithoutVat
+     * @param string $vatRate
+     * @param int $quantity
+     * @return array
+     */
+    protected function getSerializedPriceConvertedToDomainDefaultCurrency(
```

Comment:
So, when I write test, I need to use `getSerializedPriceConvertedToDomainDefaultCurrency` for output values and `getMutationPriceConvertedToDomainDefaultCurrency` for input values?

Reply:
Exactly. Maybe we could have it documented, huh?

Reply:
Ok, created issue #2081